readium/streamer/build.gradle.kts

```diff
@@ -27,26 +27,31 @@ android {
         unitTests.isIncludeAndroidResources = true
     }
     kotlinOptions {
-        freeCompilerArgs = freeCompilerArgs + "-Xopt-in=kotlin.RequiresOptIn"
-        allWarningsAsErrors = true
```

Comment:
Removing allWarningsAsErrors because compiler is throwing errors due to some deprecation warnings